content/documentation/install-uninstall.md

```diff
@@ -6,61 +6,57 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
 ---
 
 {{< q-class "box warning" >}}
-- Quire is in a closed beta. Installation will not work unless you have [requested and been given access](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
+- Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open source software. For a free license to use Quire for your publication projects (and for additional access to support from Getty staff) [please sign up](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
 {{< /q-class >}}
 
 ## macOS Installation
 
-Open your Terminal {{< q-def "command-line shell" >}} (found in your Applications/Utilities folder or by pressing Command–Space Bar and type “Terminal” ) and follow the steps below to first install the support software for Quire, and then Quire itself. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
+Installing and running Quire requires using the Terminal {{< q-def "command-line shell" >}}. Open it from your Applications/Utilities folder or by pressing Command–Space Bar and typing “Terminal”. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
 
-1. **Apple’s Xcode** is a set of developer tools for your mac. More information can be found [here](https://en.wikipedia.org/wiki/Xcode). Install Apple's Xcode by copying and pasting the following command and pressing enter. If Xcode is not already installed, an additional alert notification will pop up. Click “Install” and follow the prompts.
```

Comment:
I should double check with Matthew but this is correct, it's no longer needed. We were only having them install xcode to give them access to the `git` command in Terminal so that they could run `git clone https://github.com/thegetty/quire-cli.git`.